### ISSUE TITLE
Add fitness report for `Mageck` output

### ISSUE THREAD

**Issue by m-jahn** [MPUSP/nf-core-crispriscreen#20]
### Description of feature

- currently two reports are saved as ouput, 1x for counts, and 1x for Deseq2 fitness
- with new Mageck module, also need 1x report for Mageck output or integrate QC/analysis in existing fitness report (.Rmd)

**Comment by m-jahn** [comment]
@ute-hoffmann are you using the Mageck output? Would it make sense for you to add fitness report for Mageck?

**Comment by ute-hoffmann** [comment]
I'm currently usually not using Mageck, so not of real importance to me

**Comment by m-jahn** [comment]
OK good to know. Because it is too slow, or not informative?

**Comment by ute-hoffmann** [comment]
Half of the projects I am working on are enzyme engineering projects where we do not have several sgRNAs targeting the same gene. Just realized that this might change for a larger library we'll work on, with several barcodes associated with one gene. So might be of interest.
And then, I think it crashed too frequently and I still wanted to read a bit more about the output it gives. And I always forgot to do so. But guess Mageck is probably giving a better analysis than the usual tests

**Comment by m-jahn** [comment]
>And then, I think it crashed too frequently and I still wanted to read a bit more about the output it gives. And I always forgot to do so. But guess Mageck is probably giving a better analysis than the usual tests

I'm not sure about this. People use it but it seems it's not further developed apart from bug fixes maybe. And the performance is lousy. The DESeq and edger packages on the other hand are not made for this purpose but seem to do the task well

**Comment by ute-hoffmann** [comment]
Hmm yeah I thought more about the adjusted p values assigned to different genes. With the Wilcoxon test, I always get pretty lousy adjusted p values and people claim one should only use it for more than (10?) replicates or so. Mageck might help with that

**Comment by m-jahn** [comment]
yes that's an issue. the rank sum test is not ideal for significance analysis, it produces too discrete (non-continuous) distributions. if you stumble over an alternative I'd be happy to hear.
